packages/e2e-tests/config/flaky-tests-reporter.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * A **flaky** test is defined as a test which failed but eventually passed.
```

Comment:
Should we give a bit more context here, e.g. explain that a test that initially fails is given 2 re-runs and marked as flaky if at least one of those re-runs passes? Or something, I'm not sure I got this right 😅

Reply:
Yeah, sure! I'll provide more context like in https://github.com/WordPress/gutenberg/pull/34432#discussion_r699862384.